.travis.sh

```diff
@@ -24,6 +24,7 @@ function configure()
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-Debug} \
       -DGTSAM_BUILD_TESTS=${GTSAM_BUILD_TESTS:-OFF} \
       -DGTSAM_BUILD_UNSTABLE=${GTSAM_BUILD_UNSTABLE:-ON} \
+      -DGTSAM_WITH_TBB=${GTSAM_WITH_TBB:-ON} \
```

Comment:
Would it be better to set the default to `OFF` and then set the flag to `ON` where required for the sake of brevity and readability?

Reply:
I was thinking that but then I saw that GTSAM_BUILD_UNSTABLE was also like that so I didn't. I'll go ahead and make the change.

Reply:
Let me know if you want me to make GTSAM_BUILD_UNSTABLE to be default to OFF and I can change that as well. Or I can make another PR for it later.

Reply:
Let's keep `GTSAM_BUILD_UNSTABLE` as is for now.